src/trait_manager.rs

```diff
@@ -61,7 +111,7 @@ impl TraitManager {
                 );
                 Ok(OAMTrait::Ingress(ing))
             }
-            traits::VOLUME_MOUNTER => {
+           traits::VOLUME_MOUNTER => {
```

Comment:
indent not correct?

Reply:
I started with enum as it is recommended way in Rust but I found it restrictive. For example, named struct cannot be added without changing the schema or if anyone wants to add a new trait, they didn't have to update that enum. So I decided on parsing as JSON value which could be interpreted by the trait implementer.

I'll fix the indentation bug.